source/_posts/2016-09-15-reinventing-our-continuous-learning-practice.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Reinventing our continuous learning practice
+comments: true
+author:
+  name: Sebastián Martínez
+  email: sebastian@wyeworks.com
+  twitter_handle: smartinez87
+  github_handle:  smartinez87
+  image:  /images/team/sebastian-martinez.jpg
+  description: VP of Engineering at WyeWorks. Ruby on Rails hacker. ExceptionNotification maintainer. Coffee & bacon lover.
+published: true
+---
+
+A bit over a year ago I wrote [this article](https://wyeworks.com/blog/2015/7/16/technical-thursdays-or-how-we-do-continuous-learning/) about continuous learning and
+talked about our Technical Thursdays. If you haven't read it yet, I
+encourage you to do so! In there I talked about the importance of having
+a great team that is curious and passionate about learning.
+
```

Comment:
We need to add `<!--more-->` somewhere to mark what part of the text is shown in the main page.

It could be placed right after the first paragraph, at least. Maybe we can rephrase a little bit to present the main topic right there (mentioning the need for a change in our technical thursdays)